.test_database_urls/mongodb_4

```diff
@@ -0,0 +1,2 @@
+export TEST_DATABASE_URL="mongodb://prisma:prisma@localhost:27017/"
+set -e TEST_SHADOW_DATABASE_URL
```

Comment:
Does `unset` work on your setup? We should standardize on that if it does.

Reply:
```
❯ unset
unset: command not found
```

Reply:
I just tried this, but on bash and zsh `set -e` seems to work. Do you have some weird version of your shell that breaks here?

Reply:
It doesn't always crash, but it doesn't work:

```
erewhon.tom.~/src/prisma-engines λ export ABCD=1
erewhon.tom.~/src/prisma-engines λ echo $ABCD   
1
erewhon.tom.~/src/prisma-engines λ set -e ABCD
erewhon.tom.~/src/prisma-engines λ echo $ABCD 
1
erewhon.tom.~/src/prisma-engines λ unset ABCD
erewhon.tom.~/src/prisma-engines λ echo $ABCD

erewhon.tom.~/src/prisma-engines λ 
```

My zsh config is pretty vanillay.